Tallies/TallyClerks/absorptionClerk_class.f90

```diff
@@ -0,0 +1,322 @@
+module absorptionClerk_class
+
+  use numPrecision
+  use tallyCodes
+  use genericProcedures,          only : fatalError
+  use dictionary_class,           only : dictionary
+  use particle_class,             only : particle, particleState
+  use outputFile_class,           only : outputFile
+  use scoreMemory_class,          only : scoreMemory
+  use tallyClerk_inter,           only : tallyClerk, kill_super => kill
+
+  ! Nuclear Data interface
+  use nuclearDatabase_inter,      only : nuclearDatabase
+
+  ! Tally Filters
+  use tallyFilter_inter,          only : tallyFilter
+  use tallyFilterFactory_func,    only : new_tallyFilter
+
+  ! Tally Maps
+  use tallyMap_inter,             only : tallyMap
+  use tallyMapFactory_func,       only : new_tallyMap
+
+  ! Tally Responses
+  use tallyResponseSlot_class,    only : tallyResponseSlot
+
+
+  use tallyResult_class,     only : tallyResult, tallyResultEmpty
+
+  implicit none
+  private
+
+  !!
+  !! Colision estimator of reaction rates
```

Comment:
This could be turned into a simpler analogClerk (which we need anyway!). Can remove references to collision estimators and simplify relevant bits of the code.